examples/hello_workspaces/Scarb.toml

```diff
@@ -0,0 +1,22 @@
+[workspace]
+members = [
+    "crates/**",
```

Comment:
does Cargo support globs here? 😱

Reply:
https://doc.rust-lang.org/cargo/reference/workspaces.html#the-members-and-exclude-fields

Reply:
first time I see it live actually, interesting. let's support them then if you did the work